py2hwsw/lib/hardware/basic_tests/iob_and/iob_and.py

```diff
@@ -6,6 +6,18 @@
 
 # FIXME: API no longer exists
 # import py2hwsw_api as py2hwsw
+import os
```

Comment:
delete this. It has no info. Run core descritptions using py2hwsw_shell in batch mode as discussed

Reply:
batch mode is just using < to read the file instead of typing